org.eclipse.jdt.core.compiler.batch/src/org/eclipse/jdt/core/compiler/IProblem.java

```diff
@@ -2186,11 +2186,8 @@ public interface IProblem {
 	int ExportingForeignPackage = ModuleRelated + 1320;
 	/** @since 3.42*/
 	int ModifierOnRequiresJavaBase = ModuleRelated + 1321;
-	/**
-	 * @since 3.42
-	 * @noreference related to preview feature module imports
-	 */
-	int ModifierOnRequiresJavaBasePreview = ModuleRelated + 1322;
+	/** @since 3.44 */
+	int ModifierTransitiveOnRequiresJavaBaseBelow25 = ModuleRelated + 1322;
```

Comment:
This requires extra API filter, as the public constant is renamed, it seems "noreference" is not working for such API change.


 ```
    <resource path="src/org/eclipse/jdt/core/compiler/IProblem.java" type="org.eclipse.jdt.core.compiler.IProblem">
        <filter id="405864542">
            <message_arguments>
                <message_argument value="org.eclipse.jdt.core.compiler.IProblem"/>
                <message_argument value="ModifierOnRequiresJavaBasePreview"/>
            </message_arguments>
        </filter>
    </resource>
```